pygmt/datasets/earth_relief.py

```diff
@@ -38,6 +38,12 @@ def load_earth_relief(resolution="01d", region=None, registration=None):
         ``'03m'``, ``'02m'``, ``'01m'``, ``'30s'``, ``'15s'``, ``'03s'``,
         or ``'01s'``.
 
+    use_srtm : bool or str
```

Comment:
use_strm is a boolean parameter. It shouldn't accept *str*.
```suggestion
    use_srtm : bool
```